### PR TITLE
Readds stocks balancing with less abuse [WIP]

### PR DESCRIPTION
**Pull request by ma44**

### CODE REVIEW

**Comment by GunHog** [comment]
You really need to consider using the desktop Github, your changes are failing Travis as well, you really must try these things out locally instead of blindly uploading code.

**Comment by kevinz000** [comment]
Wew

**Comment by nullbear** [comment]
You should really only make PR's when you have a working (or at least semi-working) feature.

**Comment by kevinz000** [comment]
Why is he merging from master reee.

**Comment by CyberbossJHCB** [comment]
> Merged from master (I can forgive that)
> Travis fail
> No changelog
> No desc
> Default commit titles

Come on man... Check up on https://github.com/tgstation/tgstation/blob/master/CONTRIBUTING.md

**Comment by Jalleo** [comment]
@CyberbossJHCB its sadly more of a guideline than enforced. I hope @Optimumtact proves me wrong.

**Comment by ExcessiveUseOfCobblestone** [comment]
I told him i'd remove his cargo ban if borrowing was fixed.

File: code/modules/stock_market/computer.dm [hunk 1]
```diff
@@ -6,6 +6,7 @@
 	icon_keyboard = "no_keyboard"
 	var/logged_in = "Cargo Department"
 	var/vmode = 1
+	var/borrow = 0 //Making it prevent borrowing abuse and stuff
```
Comment (optimumtact) [inline code/modules/stock_market/computer.dm]: name it something more descriptive like current_borrow_amounts

You also never decrement this value, so after you have borrowed 2 times you will never be able to borrow again during a round

**Comment by optimumtact** [comment]
@Jalleo what about it isn't enforced? You will have to meet most of the criteria in the contribution guidelines for us to consider merging your pr

**Comment by ma44** [comment]
making memes; temp closed (forever)

**Pull request closed without merging**